dist/stat_data.py

```diff
@@ -375,7 +375,9 @@ def __init__(self, name, desc, flags=''):
     CursorStat('cursor_reset', 'cursor reset calls'),
     CursorStat('cursor_restart', 'cursor operation restarted'),
     CursorStat('cursor_search', 'cursor search calls'),
+    CursorStat('cursor_search_hs', 'cursor search history store calls'),
     CursorStat('cursor_search_near', 'cursor search near calls'),
+    CursorStat('cursor_skip_hs_cur_position', 'Total number of entries skipped to position the hs cursor'),
```

Comment:
`hs` should be `history store` in the text. Similarly below.

Reply:
Fixed.